yarn.lock

```diff
@@ -1004,20 +1004,6 @@
     js-sha256 "^0.9.0"
     tweetnacl "1.0.3"
 
-"@dialectlabs/web3@0.1.6":
```

Comment:
Why was this removed from yarn.lock?

Reply:
I have updated everything to 0.1.11, so this version not used anymore

Reply:
ah, perfect. Thanks!